.github/workflows/label.yml

```
@@ -8,5 +8,6 @@ jobs:
 
     steps:
     - uses: actions/labeler@v2
+      if: contains(github.repository, "NiklasMerz/cordova-plugin-fingerprint-aio")
```

```suggestion
      if: contains(github.repository, 'NiklasMerz/cordova-plugin-fingerprint-aio')
```

Just those are supported? Strange

---

Sorry